srcpkgs/libspng/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'libspng'
+pkgname=libspng
+version=0.7.4
+revision=1
+build_style=meson
+hostmakedepends="pkg-config cmake"
+makedepends="zlib-devel"
+short_desc="Simple, modern libpng alternative"
+maintainer="eoli3n <jonathan.kirszling@runbox.com>"
+license="BSD-2-Clause"
+homepage="https://libspng.org"
+distfiles="https://github.com/randy408/libspng/archive/refs/tags/v${version}.tar.gz"
+checksum=47ec02be6c0a6323044600a9221b049f63e1953faf816903e7383d4dc4234487
+
+post_install() {
+	vlicense LICENSE
+}
```

Comment:
also this should also have a subpackage, which can easily be added with `xnew -a libspng libspng-devel`
```suggestion
}

libspng-devel_package() {
	depends="${sourcepkg}>=${version}_${revision}"
	short_desc+=" - development files"
	pkg_install() {
		vmove usr/include
		vmove usr/lib/pkgconfig
		vmove "usr/lib/*.so"
	}
}
```